### PR TITLE
[Shaman][Restoration] 10.2 - Fix Resurgence mana returns

### PR DESCRIPTION
**Pull request by BrocoliCosmique**
Update Resurgence mana returns from 10.2

### CODE REVIEW

File: WoWAnalyzer
Comment (emallson) [inline WoWAnalyzer:1]: this (accidental?) submodule needs to be removed. if you're not sure how i can pull the branch locally and delete it

**Review by emallson — COMMENTED** [review]
other than the submodule thing, looks good

File: WoWAnalyzer
Comment (BrocoliCosmique) [reply WoWAnalyzer:1]: Is that what the 3rd file mentioned is ? I have no idea how I did that, I must have botched a commit since I'm still new to git.

Comment (emallson) [reply WoWAnalyzer:1]: yea, submodules tend to be a PITA to deal with so i'll get on tomorrow when im doing a bunch of wowa stuff and clean that up + merge

**Pull request merged**